libretro-common/include/libretro.h

```diff
@@ -1,8 +1,13 @@
-/* Copyright (C) 2010-2020 The RetroArch team
+/**
+ * libretro.h is a simple API that allows for the creation of games and emulators.
+ *
+ * @file libretro.h
+ * @version 1
+ * @author libretro
+ * @copyright Copyright (C) 2010-2020 The RetroArch team
```

Comment:
Don't forget to update the copyright to 2023.

Reply:
``` suggestion
 * @copyright Copyright (C) 2010-2023 The RetroArch team
 * ```